spec/lib/test_boosters/file_exclude_spec.rb

```diff
@@ -3,50 +3,129 @@
 describe 'Excluding a path from running when' do
   subject { distributor.all_files.any? { |f| f.include? banished_dir } }
```

Comment:
Maybe fine tune this by using a named subject such as `test_files_present` so your specs can read:
`expect(test_files_present).to be true`.